app/database/accessor/breeze_buddy/rag_embeddings.py

```
@@ -0,0 +1,171 @@
+        embeddings: Float32 numpy array of shape (N, 1536).
+        metadata: Per-chunk metadata dicts (must contain ``source`` key).
+
+    Returns:
+        Number of chunks upserted.
+    """
+    if not texts:
+        return 0
+
+    n = len(texts)
+    chunks: List[Tuple[int, str, str, List[float]]] = []
+    for i, (text, emb, meta) in enumerate(zip(texts, embeddings, metadata)):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)